frontend/src/components/Forms/AddPermitForm.js

```diff
@@ -0,0 +1,196 @@
+import React from "react";
+import { connect } from "react-redux";
+import { compose } from "redux";
+import PropTypes from "prop-types";
+import { Field, reduxForm, formValueSelector } from "redux-form";
+import RenderField from "@/components/common/RenderField";
+import RenderSelect from "@/components/common/RenderSelect";
+import RenderDate from "@/components/common/RenderDate";
+import { Form, Button, Col, Row, Popconfirm, Tabs } from "antd";
+import * as FORM from "@/constants/forms";
+import { required, dateNotInFuture } from "@/utils/Validate";
+import { resetForm } from "@/utils/helpers";
+import { getDropdownPermitStatusOptions } from "@/selectors/staticContentSelectors";
+import CustomPropTypes from "@/customPropTypes";
+
+const propTypes = {
+  handleSubmit: PropTypes.func.isRequired,
+  closeModal: PropTypes.func.isRequired,
+  permitStatusOptions: PropTypes.arrayOf(CustomPropTypes.dropdownListItem).isRequired,
+  title: PropTypes.string.isRequired,
+  submitting: PropTypes.bool.isRequired,
+  permitTypeCode: PropTypes.string,
+  permitActivityTypeCode: PropTypes.string,
+};
+
+const defaultProps = {
+  permitTypeCode: "",
+  permitActivityTypeCode: "",
+};
+
+const permitTypes = [
+  {
+    label: "Coal",
+    value: "C",
+  },
+  {
+    label: "Mineral",
+    value: "M",
+  },
+  {
+    label: "Placer",
+    value: "P",
+  },
+  {
+    label: "Sand & Gravel",
+    value: "G",
+  },
+  {
+    label: "Quarry",
+    value: "Q",
+  },
+];
+
+const permitActivityTypes = [
+  {
+    label: "Operation",
+    value: "",
+  },
+  {
+    label: "Exploration",
+    value: "X",
+  },
+];
+
+const selector = formValueSelector(FORM.ADD_PERMIT);
+const { TabPane } = Tabs;
+
+const validate = (values) => {
+  const errors = {};
+  if (
+    values.permit_activity_type === "X" &&
+    !(values.permit_type === "C" || values.permit_type === "M")
+  ) {
+    errors.permit_activity_type = "Exploration activity is only valid for Coal and Placer permits";
+  }
+  return errors;
+};
+
+export const AddPermitForm = (props) => (
+  <Form layout="vertical" onSubmit={props.handleSubmit}>
+    <Tabs
+      defaultActiveKey="1"
+      onChange={(activeKey) => {
+        // validate
```

Comment:
Is this a TODO?

Reply:
I actually removed this before seeing your comment, as the tabs were placeholder/wip for upload, which won't end up using tabs anyways.